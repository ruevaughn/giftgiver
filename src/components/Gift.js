import React, { Component } from 'react';

class Gift extends Component {
  render() {
    return (
      <div></div>
    );
  }
}

export default Gift;
