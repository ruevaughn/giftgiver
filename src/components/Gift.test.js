import React from 'react';
import { shallow } from 'enzyme';
import Gift from './Gift';

describe('Gift', () => {
  const gift = shallow(<Gift />);

  it('renders properly', () => {
    expect(gift).toMatchSnapshot();
  });
});
