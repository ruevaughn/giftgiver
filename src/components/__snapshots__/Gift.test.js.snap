// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gift renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Gift />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Form
        bsClass="form"
        componentClass="form"
        horizontal={false}
        inline={false}
>
        <FormGroup
                bsClass="form-group"
        >
                <ControlLabel
                        bsClass="control-label"
                        srOnly={false}
                >
                        Person
                </ControlLabel>
                <FormControl
                        bsClass="form-control"
                        className="input-person"
                        componentClass="input"
                        onChange={[Function]}
                />
        </FormGroup>
</Form>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsClass": "form",
        "children": <FormGroup
          bsClass="form-group"
>
          <ControlLabel
                    bsClass="control-label"
                    srOnly={false}
          >
                    Person
          </ControlLabel>
          <FormControl
                    bsClass="form-control"
                    className="input-person"
                    componentClass="input"
                    onChange={[Function]}
          />
</FormGroup>,
        "componentClass": "form",
        "horizontal": false,
        "inline": false,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "form-group",
          "children": Array [
            <ControlLabel
              bsClass="control-label"
              srOnly={false}
>
              Person
</ControlLabel>,
            <FormControl
              bsClass="form-control"
              className="input-person"
              componentClass="input"
              onChange={[Function]}
/>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "control-label",
              "children": "Person",
              "srOnly": false,
            },
            "ref": null,
            "rendered": "Person",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "form-control",
              "className": "input-person",
              "componentClass": "input",
              "onChange": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Form
          bsClass="form"
          componentClass="form"
          horizontal={false}
          inline={false}
>
          <FormGroup
                    bsClass="form-group"
          >
                    <ControlLabel
                              bsClass="control-label"
                              srOnly={false}
                    >
                              Person
                    </ControlLabel>
                    <FormControl
                              bsClass="form-control"
                              className="input-person"
                              componentClass="input"
                              onChange={[Function]}
                    />
          </FormGroup>
</Form>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "form",
          "children": <FormGroup
            bsClass="form-group"
>
            <ControlLabel
                        bsClass="control-label"
                        srOnly={false}
            >
                        Person
            </ControlLabel>
            <FormControl
                        bsClass="form-control"
                        className="input-person"
                        componentClass="input"
                        onChange={[Function]}
            />
</FormGroup>,
          "componentClass": "form",
          "horizontal": false,
          "inline": false,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "form-group",
            "children": Array [
              <ControlLabel
                bsClass="control-label"
                srOnly={false}
>
                Person
</ControlLabel>,
              <FormControl
                bsClass="form-control"
                className="input-person"
                componentClass="input"
                onChange={[Function]}
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "control-label",
                "children": "Person",
                "srOnly": false,
              },
              "ref": null,
              "rendered": "Person",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "form-control",
                "className": "input-person",
                "componentClass": "input",
                "onChange": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
